easybuild/easyconfigs/f/fastqc/fastqc-0.11.4-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'EB_Java'
```

Comment:
The only thing that `EB_Java` does on top of `PackedBinary` is to update `$JAVA_HOME`.

Using the `fastqc` install prefix to define `$JAVA_HOME` is probably very wrong?

So, `PackedBinary` should be used here instead (and in other `fastqc` easyconfigs)?